zynqmp/config.patch

```diff
@@ -0,0 +1,14 @@
+--- config	2018-06-15 13:08:27.000000000 +0200
```

Comment:
I'm a bit uneasy doing patching of files on the fly. What is the reason for this not being submitted directly to the upstream tree?

Reply:
Not really a patch. I would have to ship a config file for each supported board without this trick. Indeed, config have the same structure but different contents.

Reply:
Patch replaced by use of sed command